<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QMainWindow" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToolBar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MainWidget" name="mainWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>882</width>
        <height>468</height>
       </size>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-70</x>
         <y>260</y>
         <width>1311</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1,0,2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="nbNodesLabel">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Nombre de nœuds: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nbEdgesLabel">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Nombre d'arêtes: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nodeWeightLabel">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Pondération du nœud choisi: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nbNodesSelected">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Nombre de nœuds sélectionnés: </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>124</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>106</width>
            <height>25</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveauTableauAleatoire"/>
    <addaction name="actionOuvrirTableau"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionEnregistrerSous"/>
    <addaction name="actionExporter"/>
    <addaction name="actionApercuAvantImpression"/>
    <addaction name="actionImprimer"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRestaurer"/>
    <addaction name="separator"/>
    <addaction name="actionToutSelectionner"/>
    <addaction name="separator"/>
    <addaction name="actionCopier"/>
    <addaction name="separator"/>
    <addaction name="actionFusionner"/>
    <addaction name="actionChoisirCouleurs"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionZoomPlus"/>
    <addaction name="actionZoomMoins"/>
    <addaction name="actionImageTailleReelle"/>
    <addaction name="actionPleinEcran"/>
    <addaction name="separator"/>
    <addaction name="actionBarreDeStatus"/>
    <addaction name="actionBarreDOutils"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionModeSombre"/>
    <addaction name="separator"/>
    <addaction name="actionLangues"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionManuelAide"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveauTableauAleatoireTool"/>
   <addaction name="actionOuvrirTool"/>
   <addaction name="actionEnregistrerTool"/>
   <addaction name="actionEnregistrerSousTool"/>
   <addaction name="actionExporterTool"/>
   <addaction name="actionAnnulerTool"/>
   <addaction name="actionRestaurerTool"/>
   <addaction name="actionZoomPlusTool"/>
   <addaction name="actionZoomMoinsTool"/>
   <addaction name="actionFusionTool"/>
   <addaction name="actionChoisirCouleurTool"/>
  </widget>
  <widget class="QStatusBar" name="barreDeStatut"/>
  <action name="actionNouveauTableauAleatoire">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-ajouter-48.png</normaloff>../Icones/icons8-ajouter-48.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau tableau aleatoire...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOuvrirTableau">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-ajouter-un-onglet-48(1).png</normaloff>../Icones/icons8-ajouter-un-onglet-48(1).png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir tableau...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-sauvegarder-48.png</normaloff>../Icones/icons8-sauvegarder-48.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-image-entière-48.png</normaloff>../Icones/icons8-image-entière-48.png</iconset>
   </property>
   <property name="text">
    <string>Exporter...</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="text">
    <string>Imprimer...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-annuler-48.png</normaloff>../Icones/icons8-annuler-48.png</iconset>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRestaurer">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-refaire-48.png</normaloff>../Icones/icons8-refaire-48.png</iconset>
   </property>
   <property name="text">
    <string>Restaurer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionZoomPlus">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-zoomer-48.png</normaloff>../Icones/icons8-zoomer-48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom avant</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomMoins">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-dézoomer-48.png</normaloff>../Icones/icons8-dézoomer-48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom arriere</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionPleinEcran">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plein ecran</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionBarreDeStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre de statuts</string>
   </property>
  </action>
  <action name="actionBarreDOutils">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre d'outils</string>
   </property>
  </action>
  <action name="actionModeSombre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode sombre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLangues">
   <property name="text">
    <string>Langues...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A propos...</string>
   </property>
  </action>
  <action name="actionToutSelectionner">
   <property name="text">
    <string>Tout selectionner</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAnnulerTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-annuler-48.png</normaloff>../Icones/icons8-annuler-48.png</iconset>
   </property>
   <property name="text">
    <string>Annuler Tool</string>
   </property>
   <property name="toolTip">
    <string>Annuler</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRestaurerTool">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-refaire-48.png</normaloff>../Icones/icons8-refaire-48.png</iconset>
   </property>
   <property name="text">
    <string>Restaurer Tool</string>
   </property>
   <property name="toolTip">
    <string>Restaurer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomPlusTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-zoomer-48.png</normaloff>../Icones/icons8-zoomer-48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool Plus</string>
   </property>
   <property name="toolTip">
    <string>Zoomer</string>
   </property>
  </action>
  <action name="actionZoomMoinsTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-dézoomer-48.png</normaloff>../Icones/icons8-dézoomer-48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool Moins</string>
   </property>
   <property name="toolTip">
    <string>Dézoomer</string>
   </property>
  </action>
  <action name="actionFusionTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-pincer-48.png</normaloff>../Icones/icons8-pincer-48.png</iconset>
   </property>
   <property name="text">
    <string>Fusion Tool</string>
   </property>
   <property name="toolTip">
    <string>Fusionner 2 noeuds du graphe</string>
   </property>
  </action>
  <action name="actionOuvrirTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-ajouter-un-onglet-48(1).png</normaloff>../Icones/icons8-ajouter-un-onglet-48(1).png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir Tool</string>
   </property>
   <property name="toolTip">
    <string>Ouvrir un fichier</string>
   </property>
  </action>
  <action name="actionChoisirCouleurTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-cercle-rvb-40.png</normaloff>../Icones/icons8-cercle-rvb-40.png</iconset>
   </property>
   <property name="text">
    <string>Choisir Couleur Tool</string>
   </property>
   <property name="toolTip">
    <string>Choisir la couleur du noeud sélectionné</string>
   </property>
  </action>
  <action name="actionFusionner">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-pincer-48.png</normaloff>../Icones/icons8-pincer-48.png</iconset>
   </property>
   <property name="text">
    <string>Fusionner</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionEnregistrerSous">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-enregistrer-sous-48.png</normaloff>../Icones/icons8-enregistrer-sous-48.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionImageTailleReelle">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-zoom-taille-réelle-48.png</normaloff>../Icones/icons8-zoom-taille-réelle-48.png</iconset>
   </property>
   <property name="text">
    <string>Image taille réelle </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionManuelAide">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-aide-48.png</normaloff>../Icones/icons8-aide-48.png</iconset>
   </property>
   <property name="text">
    <string>Manuel d'aide...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionApercuAvantImpression">
   <property name="text">
    <string>Aperçu avant impression</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionChoisirCouleurs">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-cercle-rvb-40.png</normaloff>../Icones/icons8-cercle-rvb-40.png</iconset>
   </property>
   <property name="text">
    <string>Changer la couleur d'un noeud</string>
   </property>
  </action>
  <action name="actionEnregistrerTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-sauvegarder-48.png</normaloff>../Icones/icons8-sauvegarder-48.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="toolTip">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEnregistrerSousTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-enregistrer-sous-48.png</normaloff>../Icones/icons8-enregistrer-sous-48.png</iconset>
   </property>
   <property name="text">
    <string>EnregistrerSous</string>
   </property>
   <property name="toolTip">
    <string>Enregistrer Sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNouveauTableauAleatoireTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-ajouter-48.png</normaloff>../Icones/icons8-ajouter-48.png</iconset>
   </property>
   <property name="text">
    <string>NouveauTableauAleatoire</string>
   </property>
   <property name="toolTip">
    <string>Générer un nouveau tableau aléatoire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExporterTool">
   <property name="icon">
    <iconset>
     <normaloff>../Icones/icons8-image-entière-48.png</normaloff>../Icones/icons8-image-entière-48.png</iconset>
   </property>
   <property name="text">
    <string>Exporter</string>
   </property>
   <property name="toolTip">
    <string>Exporter le graphe</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header location="global">mainwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
